docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Line 1: I'd call this <code>make_loss_func</code>  -- specifically it doesn't make an MSE, it makes a function. (SO or <code>make_mse_func</code>)  <span class="ql-cursor">﻿</span></p><p>Line 1: Add space after comma</p><p>Line 6: ditto</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Agreed, fixed spaces</p>